src/osd/ClassHandler.h

```diff
@@ -36,8 +36,7 @@ class ClassHandler
       return flags;
     }
     ClassMethod(const char* name, func_t call, int flags, ClassData* cls)
-      : name{name}, func{call}, flags{flags}, cls{cls}
-    {}
+      : name{name}, func{call}, flags{flags}, cls{cls} {}
```

Comment:
this change might be controversial. as we use both styles in our code base.

Reply:
I see, change abandoned.